awx/ui_next/src/components/JobList/JobListCancelButton.jsx

```diff
@@ -54,35 +53,32 @@ function JobListCancelButton({ i18n, jobsToCancel, onCancel }) {
         <div>
           {cannotCancelPermissions.length > 0 && (
             <div>
-              {i18n._(
-                '{numJobsUnableToCancel, plural, one {You do not have permission to cancel the following job:} other {You do not have permission to cancel the following jobs:}}',
-                {
-                  numJobsUnableToCancel: cannotCancelPermissions.length,
-                }
-              )}
-              {' '.concat(cannotCancelPermissions.join(', '))}
+              <Plural
+                value={cannotCancelPermissions.length}
+                one="You do not have permission to cancel the following job:"
+                other="You do not have permission to cancel the following jobs:"
+              />
             </div>
           )}
           {cannotCancelNotRunning.length > 0 && (
             <div>
-              {i18n._(
-                '{numJobsUnableToCancel, plural, one {You cannot cancel the following job because it is not running:} other {You cannot cancel the following jobs because they are not running:}}',
-                {
-                  numJobsUnableToCancel: cannotCancelNotRunning.length,
-                }
-              )}
-              {' '.concat(cannotCancelNotRunning.join(', '))}
+              <Plural
+                value={cannotCancelNotRunning.length}
+                one="You cannot cancel the following job because it is not running"
+                other="You cannot cancel the following jobs because they are not running"
+              />
             </div>
           )}
         </div>
       );
     }
     if (numJobsToCancel > 0) {
-      return i18n._(
-        '{numJobsToCancel, plural, one {Cancel selected job} other {Cancel selected jobs}}',
-        {
-          numJobsToCancel,
-        }
+      return (
+        <Plural
+          value={numJobsToCancel}
+          one={i18n._(t`Cancel selected job`)}
```

Comment:
Since v3 you don't need i18n._() when using the t`` macro.
Also in the Plural component you don't need to wrap one,other... with the t. just strings :)

Reply:
@semoal you mentioned the Plural component.  We are having some trouble with this component described here https://github.com/lingui/js-lingui/issues/1003.  Are you able to offer and guidance on how to solve that problem?

Reply:
If you need to remove all this strings I have a cool regex to do this in vscode:
```
i18n._\(t`(.*?)`\)
```

You copy this regex and will find all the occurrences with i18n._(tànystring), and then in your replace bar of the vscode you add 
```
t`$1`
```

![Screenshot 2021-04-07 at 11 47 59](https://user-images.githubusercontent.com/22656541/113846838-2d40a800-9797-11eb-8b9e-fd0d6eeac5e7.png)